File: 0001 Code Chef/0021 Biryani classes/implementation/0007 ruby/biryani_classes.rb
test = gets.chomp.to_i
while(test > 0) do
    stringInput = gets.chomp
    stringArray = stringInput.split(" ")
    x = stringArray[0].to_i
    y = stringArray[1].to_i
    total = x * y
    puts total
    test = test - 1
end

File: 0001 Code Chef/0021 Biryani classes/implementation/0004 java/BiryaniClasses.java
import java.util.Scanner;

public class BiryaniClasses{
    public static void main(String[] args){
        Scanner console = new Scanner(System.in);
        String stringInput;
        stringInput = console.nextLine();
        int test = Integer.parseInt(stringInput);
        while(test > 0){
            stringInput = console.nextLine();
            String[] stringArray = stringInput.split(" ");
            int x = Integer.parseInt(stringArray[0]);
            int y = Integer.parseInt(stringArray[1]);
            int total = x * y;
            System.out.println(total);
            test = test - 1;
        }
    }
}